src/master/production/fips.md

```diff
@@ -0,0 +1,43 @@
+# FIPS 140-2
+
+<WrappedSection>
+
+The Federal Information Processing Standard (FIPS) 140-2 publication describes United States government-approved security requirements for cryptographic modules. [FIPS-140](https://csrc.nist.gov/publications/detail/fips/140/2/final) series is a collection of computer security standards set by the National Institute of Standards and Technology (NIST) for the United States government. FIPS 140–2 defines the critical security parameters vendors must use for encryption before selling to the U.S government.
+
+For a fully FIPS-compliant deployment of immudb a few things are required:
+
+- immudb must be compiled with a FIPS validated cryptographic module
+- immudb must be configured to use FIPS-approved cryptographic algorithms
+- immudb components (immuadmin and immuclient) must be compiled with a FIPS-validated cryptographic module
+
+For immudb, adherence to FIPS 140-2 is ensured by:
+- Using FIPS approved / NIST-recommended cryptographic algorithms through the use of `goboring/golang` container image. Since the native go crypto standard library is not FIPS compliant, we use the Google-provided Go implementation that has patches on top of standard Go to enable integrating BoringCrypto. immudb components are built with this image as a build base.
```

Comment:
Please check with markdown-lint. 

E.g. here - please add an empty line before unordered list (https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md032---lists-should-be-surrounded-by-blank-lines)

Reply:
Thanks bart for the info, is there some online tool to use for linter that you recommend? I am using this now

https://codebeautify.org/markdown-formatter

Reply:
fixed this

Reply:
I'm using [markdown plugin in VS Code - it has the linter integrated.](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) plugin in VS code it's pretty good there. Was also using [mdl tool](https://github.com/markdownlint/markdownlint) before.